maplibre-winit/src/input/mod.rs

```diff
@@ -77,13 +83,17 @@ impl InputController {
                 ..
             } => {
                 if !self.shift_handler.process_key_press(*key, *state) {
-                    if !self.tilt_handler.process_key_press(*key, *state) {
-                        self.zoom_handler.process_key_press(*key, *state)
+                    if !self.debug_handler.process_key_press(*key, *state) {
```

Comment:
why not replace the whole thing with an OR condition?  I think this is identical:

```rust
self.shift_handler.process_key_press(*key, *state) ||
self.debug_handler.process_key_press(*key, *state) ||
self.zoom_handler.process_key_press(*key, *state)
```

Reply:
might make sense to push the changes before solving the conv - easier to keep track of what was done/rejected vs pending